markets/perps-market/contracts/modules/AsyncOrderModule.sol

```diff
@@ -236,7 +237,9 @@ contract AsyncOrderModule is IAsyncOrderModule {
             runtime.newPositionSize,
             totalFees,
             runtime.settlementReward,
-            trackingCode,
+            market.size,
+            market.skew,
```

Comment:
@bachstatter My original idea was to add these to this event, but I'm wondering if a cleaner solution is creating a new event for `MarketStatus`. In V2 we had to fetch this data using `eth_call` because it wasn't in an event (`marketDebt` in V2)

Reply:
kind of like your idea. We can add the event where we update those parameters (https://github.com/Synthetixio/synthetix-v3/blob/9fa0cdef61ee644d17c14d7d90ee9dab8a7b2ac8/markets/perps-market/contracts/storage/PerpsMarket.sol#L146) and will be available anytime we update it.

Reply:
> In V2 we had to fetch this data using eth_call because it wasn't in an event (marketDebt in V2)

What would you put in `MarketStatus`, that's not in the settle event? Defintaly want to figure out away so you can avoid eth_call

> kind of like your idea. We can add the event where we update those parameters and will be available anytime we update it.

`oldPosition.updatePosition(newPosition);` is only called from settle, but I guess it make sense to put it there

Reply:
> `oldPosition.updatePosition(newPosition);` is only called from settle

Actually this might be a bug, do we miss updating market.size/ market.skew on liquidations?

Reply:
@bachstatter
Definitely: `marketId`, `marketSize`, `marketSkew`, `currentFundingRate`
Potentially:  `currentFundingVelocity`, market price

In v2 I would have wanted `marketDebt` to avoid the eth_calls, but there isn't a similar value for v3. I'm still not sure how to measure the unrealized pnl of existing positions to help calculate staker pnl.

Reply:
Added